README.md

```diff
@@ -103,6 +103,7 @@ In addition to [the parameters that you can configure for the MySQL Connector/J
 | Parameter       | Value           | Required      | Description  | Default Value |
 | --------------- |:---------------:|:-------------:|:------------ | ------------- |
 |`enableClusterAwareFailover` | Boolean | No | Set to true to enable the fast failover behavior offered by the AWS JDBC Driver. Set to false for simple JDBC connections that do not require fast failover functionality. | `true` |
+|`useConnectionPlugins` | Boolean | No | Use connection plugins to execute JDBC methods. Set to false to disable this feature. **NOTE:** disabling connection plugins will also disable the failover functionality and Enhanced Failure Monitoring. | `true` |
```

Comment:
**NOTE:** Since failover functionality and Enhanced Failure Monitoring are implemented with plugins, disabling connection plugins will also disable such functionality.
?